packages/ios_platform_images/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0
```

Comment:
This should be 0.2.1. 0.3.0 is a breaking change the way Dart treats pre-1.0.0 releases.